packages/map-template/README.md

```diff
@@ -1,88 +1,114 @@
+Add attributes to the web component as needed (see supported properties below).
 
-If you have a React app, you can use any of these React Components in your own app. If you want to not use a bottom sheet for instance, you could import the `Directions` component and work with that in your app.
+# React component: Installation and usage
 
-#### Working on Components used in Map Template
+## Using NPM
 
-To have any Stencil component changes be reflected in this project, you need to run  `npx lerna run build` from the root folder. There are no watch scripts yet.
+Install the package:
 
-## Supported properties on the MapsIndoors Map component
+`npm install @mapsindoors/map-template`
 
-|Property|Type|Description|
-|:--|:--|:--|
-|`apiKey`|`string`|The MapsIndoors Solution you want to load. Takes both API key as string and "App alias". |
-|`gmApiKey`|`string`|Your Google Maps API key. |
-|`mapboxAccessToken`|`string`|Your Mapbox Access Token. Setting it will load a Mapbox map. If you set both a Mapbox Access Token and Google Maps API key, the Mapbox Access Token takes precedence. |
-|`venue`|`string`|The Venue to load from your MapsIndoors Solution. |
-|`locationId`|`string`|Set a MapsIndoors Location ID to show it on the map and its details in the sheet. |
-|`primaryColor`|`string`|The primary color to use throughout the app. |
-|`logo`|`string`|The logo to show during initial load. |
-|`appUserRoles`|`array`|A list of App User Roles to apply when loading data. Used like so: `appUserRoles={["App User Role"]}`|
-|`directionsFrom`|`string`|Set a MapsIndoors Location ID to be used as origin to instantly show directions. |
-|`directionsTo`|`string`|Set a MapsIndoors Location ID to be used as destination to instantly show directions. |
-|`externalIDs`|`array`|Array of external IDs which filters the map and shows a list of locations. Because of the way browsers work, you can not use External IDs with the `,`, `&`, `#` and `+`, character in them, as they are interpreted by the browser in a particular way. |
-|`tileStyle`|`string`|Name of Tile Style to display on the map. |
+Make sure the MapsIndoors JavaScript SDK is loaded by having this somewhere in your HTML:
 
-## Deploying Map Template to a cloud storage provider
+```html
+<script src="https://app.mapsindoors.com/mapsindoors/js/sdk/4.21.4/mapsindoors-4.21.4.js.gz"></script>
+```
 
-We often use Google Cloud Storage (GCS) for deploying small useful apps for demo purposes. This guide refers to GCS, but many of the steps are identical for AWS, Azure Blob, and the like.
+Use the `MapsIndoorsMap` component in a React component:
 
-Running the regular build command (`npm run build`), it's assumed that all links refer to the root of a domain. When you deploy to a storage bucket, you need to build the app with the bucket name preprended to all links. Vite has a build option to take care of this:
+```javascript
+import MapsIndoorsMap from '@mapspeople/map-template/dist/mapsindoors-react.es';
```

Comment:
@ammapspeople Pardon my ignorance again, but is `.es` a React thing?

Reply:
No. `.es` signifies that the file is an ECMAScript Module.